### PR TITLE
handle missing qnodes or qedges

### LINKED ISSUE #3: Query generation doesn't handle a qgraph with no edges.
If you pass an empty list in as qedges, we get an object like this back: {nodes: [...], edges: }. This is directly after the initial MATCH statement.

### PR DESCRIPTION
**Pull request by patrickkwang**
Fixes #3.

### CODE REVIEW

**Comment by patrickkwang** [comment]
Also fixes #5.

File: reasoner/cypher.py [hunk 1]
```diff
@@ -76,11 +76,11 @@ def assemble_results(qnodes, qedges, **kwargs):
     knodes = ' + '.join([
         'collect(DISTINCT `{0}`)'.format(qnode_id)
         for qnode_id in qnodes
-    ])
+    ]) or '[]'
```
Comment (maximusunc) [inline reasoner/cypher.py:79]: for readability, I would put these down on line 90 and 91 with the others.

**Comment by maximusunc** [comment]
Something to discuss: if the user passes in a qgraph that is missing nodes or edges, we aren't catching that and the application will error when trying to mapize the qgraph. Do we want some sort of check to make sure those properties are there? The error thrown is a KeyError.

**Comment by maximusunc** [comment]
👍

**Pull request merged**